### DISCUSSION TITLE
Welcome to botd Discussions!

### DISCUSSION THREAD

**Discussion by Valve** [fingerprintjs/BotD#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NJseo** [comment]
So I have integrated botD with JS on the frontend and posting to the backend to return results to the frontend. In my way of thinking, bad bots/vm should be blocked or redirected with exceptions. Unsure on how this can be implemented. Been looking into solutions for this, any ideas?

**Reply by NJseo** [reply]
It should be noted that I have never built with NPM so I am unsure if I have the skillset to set this up.

**Reply by Valve** [reply]
@NJseo  for blocking, I'd recommend a WAF product, for example https://aws.amazon.com/waf/
